package.json

```diff
@@ -193,7 +193,7 @@
     "terminal-banner": "1.1.0",
     "through": "2.3.8",
     "ts-node": "8.3.0",
-    "typescript": "3.7.4"
+    "typescript": "3.8.2"
```

Comment:
Why do we need to upgrade Typescript? Have you checked w TR1 if this upgrade could cause problems?

Reply:
A dependency we have because of testing library and prettier is `pretty-format`
In its last update it used the `export type` typescript format introduced in TS 3.8. unless w efind a way to avoid the updated of pretty-format, we will need to update typescript.
I will see with the TR1 team if they see any problem with it.